dev-workflow-ui/src/ch/ivyteam/workflowui/cases/CasesIvyDevWfBean.java

```diff
@@ -29,24 +30,18 @@ public String getName(ICase icase) {
   }
 
   public String getStateIcon(CaseState caseState) {
-    switch (caseState) {
-      case DONE:
-        return "check-circle-1 case-state-done";
-      case CREATED:
-      case RUNNING:
-        return "hourglass case-state-in-progress case-state-in-progress";
-      case DESTROYED:
-      case ZOMBIE:
-        return "alert-circle case-state-zombie-destroyed";
-      default:
-        return "synchronize-arrows";
-    }
+    return switch (caseState) {
+      case DONE -> "check-circle-1 case-state-done";
+      case CREATED, RUNNING -> "hourglass case-state-in-progress case-state-in-progress";
```

Comment:
why is here two times "case-state-in-progress"?

Reply:
seems stragen. I'll remove.